database/index.js

```diff
@@ -0,0 +1,31 @@
+var mysql = require('mysql');
+var config = require('../config/config.js');
+var db = mysql.createConnection(config);
+
+db.connect(function(error) {
+  if (error) {
+    console.log(error);
+  } else {
+    console.log('==mySQL is connected==');
+  }
+});
+
+const getPhotos = function (experienceID, callback) {
+  var id = experienceID
+  var grabAll = `SELECT * FROM photos WHERE experience_id = '${experienceID}'`;
```

Comment:
Also, if you want to try escaping with your query variable, I can show you how (instead of using the template literal).